### PR TITLE
Migrate to structured logs pkg/util/bandwidth/ and /conntrack

### PR DESCRIPTION
**Pull request by jialaijun**
in pkg/util/bandwidth/linux.go
in pkg/util/conntrack/conntrack.go
in pkg/util/coverage/coverage.go

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it:**

Ref:

- [keps/sig-instrumentation/1602-structured-logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging)

- [Structured Logging migration instructions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

**Does this PR introduce a user-facing change?:**

`Migrate some log messages to structured logging `

### CODE REVIEW

File: pkg/util/bandwidth/linux.go [hunk 1]
```diff
@@ -60,10 +60,10 @@ func NewTCShaper(iface string) Shaper {
 }
 
 func (t *tcShaper) execAndLog(cmdStr string, args ...string) error {
-	klog.V(6).Infof("Running: %s %s", cmdStr, strings.Join(args, " "))
+	klog.V(6).InfoS("Running", "cmdStr", cmdStr, "args", strings.Join(args, " "))
 	cmd := t.e.Command(cmdStr, args...)
 	out, err := cmd.CombinedOutput()
-	klog.V(6).Infof("Output from tc: %s", string(out))
+	klog.V(6).InfoS("print Output from tc", "out", string(out))
```
Comment (lala123912) [inline pkg/util/bandwidth/linux.go:66]: print Output->Print output

File: pkg/util/coverage/coverage.go [hunk 2]
```diff
@@ -87,6 +87,6 @@ func FlushCoverage() {
 	// This gets us atomic updates from the perspective of another process trying to access
 	// the file.
 	if err := os.Rename(tempCoveragePath(), coverageFile); err != nil {
-		klog.Errorf("Couldn't move coverage file from %s to %s", coverageFile, tempCoveragePath())
+		klog.ErrorS(err, "Couldn't move coverage file", "from", coverageFile, "to", tempCoveragePath())
```
Comment (shubheksha) [inline pkg/util/coverage/coverage.go:90]:
```suggestion
		klog.ErrorS(err, "Couldn't move coverage file", "sourcePath", coverageFile, "destinationPath", tempCoveragePath())
```

File: pkg/util/bandwidth/linux.go (hunk 1 above)
Comment (shubheksha) [inline pkg/util/bandwidth/linux.go:66]:
```suggestion
	klog.V(6).InfoS("print Output from tc", "printOutput", string(out))
```

**Review by shubheksha — CHANGES_REQUESTED** [review]

**Comment by shubheksha** [comment]
/retitle Migrate to structured logs pkg/util/bandwidth/ and /conntrack

**Comment by aojea** [comment]
/ok-to-test
please address the comments and we'll wait for @shubheksha lgtm

**Pull request closed without merging**